examples/criteo-example.ipynb

```diff
@@ -96,7 +96,7 @@
     }
    ],
    "source": [
-    "rmm.reinitialize(pool_allocator=True, initial_pool_size=0.8 * rmm.get_info().free)"
+    "rmm.reinitialize(pool_allocator=True, initial_pool_size=0.8 * nvt.io._device_mem_size(kind='free'))"
```

Comment:
If we're exposing this as part of our public API (by using it in our notebooks etc) - I think the _device_mem_size function shouldn't have a leading underscore.

Reply:
> If we're exposing this as part of our public API (by using it in our notebooks etc) - I think the _device_mem_size function shouldn't have a leading underscore.

Nice - I had the same thought, and was making this change :)